packages/vm/src/testing/mock.rs

```diff
@@ -100,23 +102,27 @@ impl Default for MockApi {
 }
 
 impl BackendApi for MockApi {
-    fn canonical_address(&self, human: &str) -> BackendResult<Vec<u8>> {
+    fn canonical_address(&self, input: &str) -> BackendResult<Vec<u8>> {
+        // mimicks formats like hex or bech32 where different casings are valid for one address
+        let normalized = input.to_lowercase();
+
+        // let gas_info = GasInfo::with_cost(DEFAULT_GAS_COST_CANONICALIZE);
```

Comment:
What is this comment for?

Reply:
This is the comment code for testing. Now it is no need. I'll remove it. Thank you.